### PR TITLE
Fix typos in tests/

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_common.kshlib [hunk 1]
```diff
@@ -56,7 +56,7 @@ function setup_testenv #[dtst]
 	if ! datasetexists $FS; then
 		log_must zfs create $FS
 	fi
-	# Volume test is only availible on globle zone
+	# Volume test is only available on globle zone
```
Comment (behlendorf) [inline tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_common.kshlib]: While you're here, "globle" -> "global"

Comment (Gelma) [reply tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_common.kshlib]: Done! I also do the squash!

File: tests/zfs-tests/tests/functional/cli_root/zfs_property/zfs_written_property_001_pos.ksh [hunk 2]
```diff
@@ -120,7 +120,7 @@ sync_pool
 written=$(get_prop written $TESTPOOL/$TESTFS1)
 writtenat3=$(get_prop written@snap3 $TESTPOOL/$TESTFS1)
 [[ $written -eq $writtenat3 ]] || \
-    log_fail "Written and written@ dont match $written $writtenat3"
+    log_fail "Written and written@ don't match $written $writtenat3"
```
Comment (rlaager) [inline tests/zfs-tests/tests/functional/cli_root/zfs_property/zfs_written_property_001_pos.ksh]: Do we normally use contractions in these? Should we? How about "do not"

Comment (Gelma) [reply tests/zfs-tests/tests/functional/cli_root/zfs_property/zfs_written_property_001_pos.ksh]: > Do we normally use contractions in these? Should we? How about "do not"

You tell me. If you want I can change it.

Comment (rlaager) [reply tests/zfs-tests/tests/functional/cli_root/zfs_property/zfs_written_property_001_pos.ksh]: I grepped. There are lots of contractions.

**Pull request merged**